src/gcp/cloudscheduler.ts

```diff
@@ -12,38 +14,116 @@ export interface Schedule {
     uri: string;
     httpMethod: string;
   };
+  retryConfig?: {
+    retryCount?: number;
+    maxRetryDuration?: string;
+    minBackoffDuration?: string;
+    maxBackoffDuration?: string;
+    maxDoublings?: number;
+  };
 }
 
-export function createJob(schedule: Schedule): Promise<void> {
+/*
```

Comment:
for jsdoc block comments it's usually `/**` ... `*/` on their own lines, and the `*`s line up.